api/assessment.go

```diff
@@ -73,10 +80,16 @@ func (h AssessmentHandler) List(ctx *gin.Context) {
 		_ = ctx.Error(result.Error)
 		return
 	}
+	questionnaires, err := assessment.NewQuestionnaireResolver(h.DB(ctx))
```

Comment:
The preload on line 77 should have the Questionnaire relationships on each assessment populated, and you should be able to grab the required property from there rather than setting up a resolver.